lib/matplotlib/ticker.py

```diff
@@ -2138,8 +2294,10 @@ def _decade_greater(x, base):
     return greater
 
 
-def is_close_to_int(x):
-    return abs(x - np.round(x)) < 1e-10
+def is_close_to_int(x, *, tol=None):
```

Comment:
atol (as opposed to rtol)
Not sure we need to support tol=None, could just do tol=1e-10 as default.

Reply:
I think we need `None` as default, for the `is_decade` function, or we have to set the default value in both function.

Reply:
I think setting the default in both places is just fine.  (Note that if you were to write docs for these helpers (not requesting it for this PR) you'd have to document the default in both places anyways...)

Reply:
Fixed, I set the default value in both. And I change tol to atol.